contrib/mesos/pkg/executor/service/service.go

```diff
@@ -87,6 +88,7 @@ func (s *KubeletExecutorServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.SuicideTimeout, "suicide-timeout", s.SuicideTimeout, "Self-terminate after this period of inactivity. Zero disables suicide watch.")
 	fs.IntVar(&s.ShutdownFD, "shutdown-fd", s.ShutdownFD, "File descriptor used to signal shutdown to external watchers, requires shutdown-fifo flag")
 	fs.StringVar(&s.ShutdownFIFO, "shutdown-fifo", s.ShutdownFIFO, "FIFO used to signal shutdown to external watchers, requires shutdown-fd flag")
+	fs.DurationVar(&s.LaunchGracePeriod, "launch-grace-period", s.LaunchGracePeriod, "Launch grace period after which launching tasks will be cancelled . Zero disables launch cancellation.")
```

Comment:
Don't forget to add new flags to hack/verify-flags/known-flags.txt. 

As you haven't seen an error during committing, you probably don't have the kubernetes post-commit hook installed locally. You should. Catches a lot of errors.